src/hardware/vga.cpp

```diff
@@ -70,7 +70,20 @@ void VGA_SetMode(VGAModes mode) {
 	VGA_StartResize();
 }
 
-void VGA_DetermineMode(void) {
+bool VGA_InTextMode()
+{
+	switch (vga.mode) {
+	case M_TEXT:
+	case M_HERC_TEXT:
+	case M_TANDY_TEXT:
+	case M_CGA_TEXT_COMPOSITE: return true;
+
+	default: return false;
+	}
+}
```

Comment:
This is not needed as I introduced `INT10_IsTextMode` for this a while ago. But let's not do any text-mode only special casing, as per my other comment.

Reply:
Removed.